src/patternfly/components/Sequence/docs/code.md

```diff
@@ -0,0 +1,33 @@
+## Overview
+
+Only if needed write a short description with implementation notes. Design and interaction notes are already written on the design specs, don't repeat information.
+
+For example for buttons: Always add a modifier class to add color to the button. Never use the class `.btn` on its own.
+
+## Accessibility
+
+| Attribute | Applied To | Outcome |
+| -- | -- | -- |
+| `aria-label="[landmark description]"` | `.pf-c-sequence` | Describes <sequence> landmark |
+| `aria-setsize` | `.pf-c-sequence__item` | Indicates the total number of items in the list |
+| `aria-posinset` | `.pf-c-sequence__item` | Indicates the position of an item within a list |
```

Comment:
I was hoping that `aria-setsize` and `aria-posinset` would provide some indication about which list item currently has focus, but when I tested this in JAWS and VO, nothing was announced. Did you notice any change when you added this in? At this point, I'm leaning toward not including this, until we find that there is support and it provides some benefit to users.

Reply:
In VO nothing was announced when I tested it either. I will leave it off for now